doc/application/application.rst

```diff
@@ -157,23 +158,25 @@ The Zephyr build system compiles and links all components of an application
 into a single application image that can be run on simulated hardware or real
 hardware.
 
+As described in `_getting_started_cmake`, on Linux and macOS you can choose
```

Comment:
This would be a bit better as:

```
:ref:`_getting_started_cmake`
```

Just in case the content gets moved around.

Reference:

> Using ref is advised over standard reStructuredText links to sections (like `Section title`_) because it works across files, when section headings are changed, and for all builders that support cross-references.

http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-arbitrary-locations